{"uid":"723137727c240add","name":"test_sign_in","fullName":"test.test_WebApp#test_sign_in","historyId":"af52238d7ee13a4d3d9fb66d5470d685","time":{"start":1728906583374,"stop":1728906591048,"duration":7674},"description":"\n    Sign in to the application\n    ","descriptionHtml":"<pre><code>Sign in to the application\n</code></pre>\n","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-mm6vw2\" tabindex=\"0\" type=\"submit\">...</button> is not clickable at point (180, 338). Other element would receive the click: <form class=\"MuiBox-root css-1j41l18\" novalidate=\"\">...</form>\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x55cd35e2c579 <unknown>","statusTrace":"def test_sign_in():\n        \"\"\"\n        Sign in to the application\n        \"\"\"\n        with allure.step(\"Sign in\"):\n            driver.find_element(AppiumBy.XPATH, '//input[@name=\"email\"]').send_keys(\n                \"ross@al.co.za\"\n            )\n            driver.find_element(AppiumBy.XPATH, '//input[@name=\"password\"]').send_keys(\n                \"P@ssword2\"\n            )\n            driver.find_element(AppiumBy.XPATH, \"//form\").click()\n            time.sleep(2)\n>           driver.find_element(by=AppiumBy.XPATH, value=\"//form/button\").click()\n\ntest/test_WebApp.py:89: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <appium.webdriver.errorhandler.MobileErrorHandler object at 0x7f5c2ed63790>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...ovalidate=\\\\\"\\\\\">...</form>\\\\n  (Session info: chrome=83.0.4103.106)\",\"stacktrace\":\"#0 0x55cd35e2c579 <unknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        https://www.w3.org/TR/webdriver/#errors\n        \"\"\"\n        payload = response.get('value', '')\n        if isinstance(payload, dict):\n            payload_dict = payload\n        else:\n            try:\n                payload_dict = json.loads(payload)\n            except (json.JSONDecodeError, TypeError):\n                return\n            if not isinstance(payload_dict, dict):\n                return\n        value = payload_dict.get('value')\n        if not isinstance(value, dict):\n            return\n        error = value.get('error')\n        if not error:\n            return\n    \n        message = value.get('message', error)\n        stacktrace = value.get('stacktrace', '')\n        # In theory, we should also be checking HTTP status codes.\n        # Java client, for example, prints a warning if the actual `error`\n        # value does not match to the response's HTTP status code.\n        exception_class: Type[sel_exceptions.WebDriverException] = ERROR_TO_EXC_MAPPING.get(\n            error, sel_exceptions.WebDriverException\n        )\n        if exception_class is sel_exceptions.WebDriverException and message:\n            if message == 'No such context found.':\n                exception_class = appium_exceptions.NoSuchContextException\n            elif message == 'That command could not be executed in the current context.':\n                exception_class = appium_exceptions.InvalidSwitchToTargetException\n    \n        if exception_class is sel_exceptions.UnexpectedAlertPresentException:\n            raise sel_exceptions.UnexpectedAlertPresentException(\n                msg=message,\n                stacktrace=format_stacktrace(stacktrace),\n                alert_text=value.get('data'),\n            )\n>       raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-mm6vw2\" tabindex=\"0\" type=\"submit\">...</button> is not clickable at point (180, 338). Other element would receive the click: <form class=\"MuiBox-root css-1j41l18\" novalidate=\"\">...</form>\nE         (Session info: chrome=83.0.4103.106)\nE       Stacktrace:\nE       #0 0x55cd35e2c579 <unknown>\n\n../../../.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:125: ElementClickInterceptedException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"\n    Sign in to the application\n    ","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-mm6vw2\" tabindex=\"0\" type=\"submit\">...</button> is not clickable at point (180, 338). Other element would receive the click: <form class=\"MuiBox-root css-1j41l18\" novalidate=\"\">...</form>\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x55cd35e2c579 <unknown>","statusTrace":"def test_sign_in():\n        \"\"\"\n        Sign in to the application\n        \"\"\"\n        with allure.step(\"Sign in\"):\n            driver.find_element(AppiumBy.XPATH, '//input[@name=\"email\"]').send_keys(\n                \"ross@al.co.za\"\n            )\n            driver.find_element(AppiumBy.XPATH, '//input[@name=\"password\"]').send_keys(\n                \"P@ssword2\"\n            )\n            driver.find_element(AppiumBy.XPATH, \"//form\").click()\n            time.sleep(2)\n>           driver.find_element(by=AppiumBy.XPATH, value=\"//form/button\").click()\n\ntest/test_WebApp.py:89: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n../../../.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <appium.webdriver.errorhandler.MobileErrorHandler object at 0x7f5c2ed63790>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...ovalidate=\\\\\"\\\\\">...</form>\\\\n  (Session info: chrome=83.0.4103.106)\",\"stacktrace\":\"#0 0x55cd35e2c579 <unknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        https://www.w3.org/TR/webdriver/#errors\n        \"\"\"\n        payload = response.get('value', '')\n        if isinstance(payload, dict):\n            payload_dict = payload\n        else:\n            try:\n                payload_dict = json.loads(payload)\n            except (json.JSONDecodeError, TypeError):\n                return\n            if not isinstance(payload_dict, dict):\n                return\n        value = payload_dict.get('value')\n        if not isinstance(value, dict):\n            return\n        error = value.get('error')\n        if not error:\n            return\n    \n        message = value.get('message', error)\n        stacktrace = value.get('stacktrace', '')\n        # In theory, we should also be checking HTTP status codes.\n        # Java client, for example, prints a warning if the actual `error`\n        # value does not match to the response's HTTP status code.\n        exception_class: Type[sel_exceptions.WebDriverException] = ERROR_TO_EXC_MAPPING.get(\n            error, sel_exceptions.WebDriverException\n        )\n        if exception_class is sel_exceptions.WebDriverException and message:\n            if message == 'No such context found.':\n                exception_class = appium_exceptions.NoSuchContextException\n            elif message == 'That command could not be executed in the current context.':\n                exception_class = appium_exceptions.InvalidSwitchToTargetException\n    \n        if exception_class is sel_exceptions.UnexpectedAlertPresentException:\n            raise sel_exceptions.UnexpectedAlertPresentException(\n                msg=message,\n                stacktrace=format_stacktrace(stacktrace),\n                alert_text=value.get('data'),\n            )\n>       raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-mm6vw2\" tabindex=\"0\" type=\"submit\">...</button> is not clickable at point (180, 338). Other element would receive the click: <form class=\"MuiBox-root css-1j41l18\" novalidate=\"\">...</form>\nE         (Session info: chrome=83.0.4103.106)\nE       Stacktrace:\nE       #0 0x55cd35e2c579 <unknown>\n\n../../../.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:125: ElementClickInterceptedException","steps":[{"name":"Sign in","time":{"start":1728906583374,"stop":1728906591047,"duration":7673},"status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class=\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-fullWidth css-mm6vw2\" tabindex=\"0\" type=\"submit\">...</button> is not clickable at point (180, 338). Other element would receive the click: <form class=\"MuiBox-root css-1j41l18\" novalidate=\"\">...</form>\n  (Session info: chrome=83.0.4103.106)\nStacktrace:\n#0 0x55cd35e2c579 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Appium_test/Appium_test/test/test_WebApp.py\", line 89, in test_sign_in\n    driver.find_element(by=AppiumBy.XPATH, value=\"//form/button\").click()\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n  File \"/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true},"afterStages":[],"labels":[{"name":"feature","value":"Test Customers Table"},{"name":"epic","value":"Customers Epic"},{"name":"parentSuite","value":"test"},{"name":"suite","value":"test_WebApp"},{"name":"host","value":"fv-az1019-265"},{"name":"thread","value":"3038-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.test_WebApp"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":1,"unknown":0,"total":2},"items":[{"uid":"e565a5b4e7571ecc","reportUrl":"https://HolmesMR-AL.github.io/Appium_test/55//#testresult/e565a5b4e7571ecc","status":"passed","time":{"start":1728906572391,"stop":1728906581259,"duration":8868}}]},"tags":[]},"source":"723137727c240add.json","parameterValues":[]}